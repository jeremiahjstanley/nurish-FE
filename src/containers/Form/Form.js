import React, { Component } from 'react';
import { connect } from 'react-redux';
import { withRouter, NavLink, Route } from 'react-router-dom';
import Search from '../Search/Search';
import FormulaContainer from '../FormulaContainer/FormulaContainer';
import './Form.css'; 
import PropTypes from 'prop-types';

class Form extends Component {
  render() {
    return (
      <div className="form-container links">
				<NavLink to='/search'>search</NavLink>
				<NavLink to='/filter'>filter</NavLink>
				<NavLink to='/formulas'>browse</NavLink>
      </div>
    );
  }
}

Form.propTypes = {
  
}

export default withRouter((Form));