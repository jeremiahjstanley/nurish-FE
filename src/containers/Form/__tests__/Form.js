import React from 'react';
import {Form} from '../Form';
import { shallow } from 'enzyme';

describe('Form', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Form />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });
});
