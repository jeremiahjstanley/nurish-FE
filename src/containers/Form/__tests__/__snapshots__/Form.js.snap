// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Form />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          aria-current="page"
          to="/search"
>
          search
</NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          to="/filter"
>
          filter
</NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          to="/formulas"
>
          browse
</NavLink>,
      ],
      "className": "form-container links",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": "search",
          "to": "/search",
        },
        "ref": null,
        "rendered": "search",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": "filter",
          "to": "/filter",
        },
        "ref": null,
        "rendered": "filter",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": "browse",
          "to": "/formulas",
        },
        "ref": null,
        "rendered": "browse",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/search"
>
            search
</NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/filter"
>
            filter
</NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/formulas"
>
            browse
</NavLink>,
        ],
        "className": "form-container links",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "search",
            "to": "/search",
          },
          "ref": null,
          "rendered": "search",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "filter",
            "to": "/filter",
          },
          "ref": null,
          "rendered": "filter",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "browse",
            "to": "/formulas",
          },
          "ref": null,
          "rendered": "browse",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
